### ISSUE TITLE
Distinguish between pressing Done and Close

### ISSUE THREAD

**Issue by skangayam** [hyakt/expo-multiple-media-imagepicker#7]
I want to use the media picker as a first part of a process. Therefore, I need to determine if the callback is invoked due to pressing Close or Done. At this time, I see the following response in both cases of pressing either Done or Close. 

How can we determine if the call back was invoked by Close or Done?

```
Promise {
  "_40": 0,
  "_55": Array [],
  "_65": 1,
  "_72": null,
}
```

**Comment by mkhoussid** [comment]
Are you awaiting for the promise?

**Comment by skangayam** [comment]
No, I have placed this component in a Modal with "Close" and "Next" buttons on the top to either close this modal (and discard the list of images selected) or move onto the next step in this process and iterate through the list of images.

I want to understand how to be able distinguish between the two actions. i.e. if the user has clicked "Close" or "Next" after selecting the images.

PS: I apologize if the question is bad. I'm a noob in ReactNative.

**Comment by ea1ik** [comment]
After experimenting some with it, the _55 value of the promise determines whether it's Done or Close. 1 is Close and 0 is Done
